scripts/addie

```diff
@@ -2896,4 +2900,4 @@ if __name__ == '__main__':
     options = parser.parse_args()
 
     # start the main program
-    main()
+    main(options)
```

Comment:
@peterfpeterson is this the best way to pass the `options` into the `MainWindow`? I was leary about doing it this way but it worked.

Reply:
I'd split up the options to pass into individual variables that are passed in.